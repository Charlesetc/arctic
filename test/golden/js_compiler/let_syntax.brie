define identity : x [
  x
]

define main [
  let x = 2
  let x = identity x
  x
]
