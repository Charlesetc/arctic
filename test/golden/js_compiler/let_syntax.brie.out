const global_functions = {1: {"Integer": fn_1_Integer},2: {}};
function fn_1_Integer(x) {
	return x;
}
function main() {
	var x = 2;
	var x = test/golden/js_compiler/let_syntax.brie_identity.partial(x, "Integer").call();
	return x;
}
var test/golden/js_compiler/let_syntax.brie_identity = new_closure(1)
var test/golden/js_compiler/let_syntax.brie_main = new_closure(2)
console.log(main());